.github/workflows/cicd-main.yml

```diff
@@ -4166,6 +4166,55 @@ jobs:
       AFTER_SCRIPT: |
         rm -f examples/asr/evaluation_transcripts.json
 
+  L2_Stable_Diffusion_Training:
+    needs: [cicd-test-container-setup]
+    uses: ./.github/workflows/_test_template.yml
+    with:
+      RUNNER: self-hosted-azure
```

Comment:
May i ask: For this test:
L2_Stable_Diffusion_Training
Do you need 1 or 2 GPUs?
Seems only 1:
trainer.devices=1
Right?
Then instead you should use:
RUNNER: self-hosted-azure-gpus-1